methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+def ending(number)
+  lyrics = <<lyrics
```

Comment:
No need to create a variable here...

    puts <<END_OF_LYRIC
    ....
    END_OF_LYRIC